src/mongoose_acc.erl

```diff
@@ -88,11 +88,12 @@
         ns_key() => Value :: any()
        }.
 
--export_type([t/0]).
+-export_type([t/0,
+              new_acc_params/0]).
 
 -type new_acc_params() :: #{
         location := location(),
-        lserver := jid:lserver(),
+        lserver => jid:lserver(),
```

Comment:
is it really optional? if so - when?

Reply:
```
-spec new_acc(state(), mongoose_acc:new_acc_params()) -> mongoose_acc:t().
new_acc(#state{host_type = HostType, server = LServer}, Params) ->
    mongoose_acc:new(Params#{host_type => HostType, lserver => LServer}).
 ```

Reply:
second argument in this spec

Reply:
ok, fixed spec instead.

Reply:
this commend should be outdated, but something is wrong with GH :)

Reply:
👍